Shifty.App/Components/ProductManager.razor

```diff
@@ -30,52 +35,53 @@
         RowStyleFunc="@_RowStyleFunc"
         FixedHeader="true"
         Height="calc(100vh - 250px)"
-        HorizontalScrollbar="false">
+        Dense="true"
+        SortMode="MudBlazor.SortMode.None">
         <Columns>
-            <TemplateColumn>
+            <TemplateColumn Title="Edit">
                 <CellTemplate>
                     <MudIconButton
-                        Size="@Size.Small" 
+                        Size="@Size.Medium"
                         Icon="@Icons.Material.Outlined.Edit"
-                        OnClick="@context.Actions.StartEditingItemAsync"/>
+                        Color="Color.Primary"
+                        OnClick="@context.Actions.StartEditingItemAsync" />
                 </CellTemplate>
             </TemplateColumn>
-            <PropertyColumn Property="x => x.Visible" Title="" IsEditable="true" InitialDirection="SortDirection.Descending">
+            <PropertyColumn Property="x => x.Visible" Title="" InitialDirection="SortDirection.Descending">
                 <CellTemplate>
                     @{
                         if (context.Item.Visible)
                         {
-                            <MudIcon Color="Color.Primary" Icon="@Icons.Material.Filled.Visibility" />
+                            <MudIcon Size="@Size.Small" Style="@($"color:{Colors.Grey.Darken3};")"
+                                Icon="@Icons.Material.Filled.Visibility" />
```

Comment:
We should use the MudBlazor Colors (like `Color.Primary`) in order to easily be able to switch designs/color schemes at a later stage. (I don't have a big opinion on which color we use, but let's define this color in our color palette. It's defined in `./Shitfy.App/Shared/MainLayout.razor`: 
```cs
readonly MudTheme _theme = new MudTheme()
{
    Palette = new Palette()
    {
        Primary = new MudColor("#775440"),
        Secondary = new MudColor("#38251a"),
        Tertiary = new MudColor("#8c674c"),
        AppbarBackground = new MudColor("#38251a"),
        Background = new MudColor("#e9e9e9"),
        Info = new MudColor("#20455f"),
        Error = new MudColor("#893c24"),
        Warning = new MudColor("#fed521"),
        Success = new MudColor("#738d4b"),
    }
};
```

We can easily extend this with a `Grey` color.
![image](https://github.com/AnalogIO/shifty-webapp/assets/95026056/4814f5b4-604f-4c3e-8ee8-ba9337e3fb2e)